.github/release-drafter.yml

```diff
@@ -0,0 +1,72 @@
+name-template: 'v$RESOLVED_VERSION'
+tag-template: 'v$RESOLVED_VERSION'
+categories:
+  - title: 'Breaking 🔨'
+    label: 'type: breaking'
```

Comment:
Do we want to have `type: ` prefix for labels? If so, we need to add new labels to the repository.

Reply:
Per your PR, the `type:` prefix exists, right? It's perhaps not necessary now, with such a short list, but could be useful to keep the prefix in case we have more labels down the road that need to be categorised among themselves.

Are you able to update the labels in the repo?

Reply:
Updated the labels!